netbox/dcim/tests/test_models.py

```diff
@@ -439,6 +453,37 @@ def test_cable_cannot_terminate_to_an_existing_connection(self):
         with self.assertRaises(ValidationError):
             cable.clean()
 
+    def test_singlepos_rearport_connections(self):
+        """
+        A RearPort with one position can be connected to anything as it is just a
+        cable extender.
```

Comment:
No such thing as a "cable extender." Be sure to explain _exactly_ what the test is doing and why it's there.